### PR TITLE
Added contributing and issue templates

### PR DESCRIPTION
**Pull request by kfichter**
Addresses #48 

These templates are up for discussion and should be reviewed before being merged.

### CODE REVIEW

File: .github/ISSUE_TEMPLATE.md [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+
+## Steps to Reproduce
+
+
+## Motivation for Change
+
+
+## System Specs
+
+- Python Version:
```
Comment (DavidKnott) [inline .github/ISSUE_TEMPLATE.md]: Can you add these two as well?
pyethereum Version: x.x.x
Environment (output of pip freeze):

File: .github/ISSUE_TEMPLATE.md [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+
+## Current Behavior
+
+
+## Expected Behavior
+
+
+## Steps to Reproduce
+
+
```
Comment (DavidKnott) [inline .github/ISSUE_TEMPLATE.md]: Can you add:
### How can it be fixed?

Fill this in if you know how to fix it.

File: .github/ISSUE_TEMPLATE.md [hunk 3]
```diff
@@ -0,0 +1,30 @@
...
+## Current Behavior
+
+
+## Expected Behavior
+
+
+## Steps to Reproduce
+
```
Comment (DavidKnott) [inline .github/ISSUE_TEMPLATE.md]: And this to steps to reproduce:
* full output of the error you received
* what command you ran
* the code that caused the failure (see [this link](https://help.github.com/articles/basic-writing-and-formatting-syntax/) for help with formatting code)

**Comment by kfichter** [comment]
@DavidKnott Added some comments + made your changes as requested.

**Pull request merged**